main.nf

```diff
@@ -1,24 +1,45 @@
 #! /usr/bin/env nextflow
 
-nextflow.enable.dsl = 2
+include { PREPARE_INPUT } from "$projectDir/subworkflows/local/prepare_input/main"
 
-include { PREPARE_INPUT } from "$projectDir/subworkflows/prepare_input/main"
+include { BUILD_DATABASES as BUILD_HIFI_DATABASES } from "$projectDir/subworkflows/local/build_databases/main"
```

Comment:
I like the inclusion with `$projectDir`, I proposed that in nf-core, but got denied...